PowerEditor/src/MISC/Exception/MiniDumper.cpp

```diff
@@ -55,11 +55,11 @@ bool MiniDumper::writeDump(EXCEPTION_POINTERS * pExceptionInfo)
 
 				if (hFile!=INVALID_HANDLE_VALUE)
 				{
-					_MINIDUMP_EXCEPTION_INFORMATION ExInfo;
+					_MINIDUMP_EXCEPTION_INFORMATION ExInfo{};
 
 					ExInfo.ThreadId = ::GetCurrentThreadId();
 					ExInfo.ExceptionPointers = pExceptionInfo;
-					ExInfo.ClientPointers = NULL;
+					ExInfo.ClientPointers = FALSE;

```

Comment:
Use `FALSE` according to documentation https://docs.microsoft.com/en-us/windows/win32/api/minidumpapiset/ns-minidumpapiset-minidump_exception_information